bubuku/zookeeper/__init__.py

```diff
@@ -197,6 +199,17 @@ def get_broker_ids(self) -> list:
         """
         return sorted(self.exhibitor.get_children('/brokers/ids'))
 
+    def get_broker_racks(self) -> Dict[int, str]:
+        """
+        Lists the rack of each broker, if it exists
+        :return: a ditionary of tuples (broker_id, rack), where rack can be None
+        """
+        brokers = sorted(self.exhibitor.get_children('/brokers/ids', include_data=True), key=lambda x: x[0])
+        racks = {}
+        for id, data in brokers:
+            racks[id] = data.get('rack')
+        return racks
```

Comment:
why sorted?
Isn't it simpler to do like 
```return {id_: data.get('rack') for id_, data in self.exhibitor.get_children('/brokers/ids', include_data=True)}```

Reply:
049f3ea